spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/support/SimpleTaskRepository.java

```diff
@@ -45,12 +46,27 @@
 
 	private boolean initialized = false;
 
+	private int maxExitMessageSize = MAX_EXIT_MESSAGE_SIZE;
+
+	private int maxTaskNameSize = MAX_TASK_NAME_SIZE;
+
+	private int maxErrorMessageSize = MAX_ERROR_MESSAGE_SIZE;
+
 	public SimpleTaskRepository(FactoryBean<TaskExecutionDao> taskExecutionDaoFactoryBean){
 		Assert.notNull(taskExecutionDaoFactoryBean, "A FactoryBean that provides a TaskExecutionDao is required");
 
 		this.taskExecutionDaoFactoryBean = taskExecutionDaoFactoryBean;
 	}
 
+	public SimpleTaskRepository(FactoryBean<TaskExecutionDao> taskExecutionDaoFactoryBean, int maxExitMessageSize,
+			int maxTaskNameSize, int maxErrorMessageSize){
```

Comment:
Thoughts about `Integer` instead of `int` so that we don't need to configure them all explicitly?